listing-about.md

```diff
@@ -1,45 +1,46 @@
 ### Meta
-- **Dataset name**: *`datasetName`*
-- **Vendor name**: *`vendorName`*
-- **Vendor Website**: *`vendorWebsite`*
+- **Dataset name**: MarketCap
+- **Vendor name**: CoinGecko
+- **Vendor Website**: https://www.coingecko.com/
 
 
 ### Introduction
 
-*`datasetName`* by *`vendorName`* *`/is/tracks`* *`assetsInMarket`* *`datasetProperty`*. The data covers *`coverageNumber`* *`coverageAssetType`*, starting in *`startDate`*, and is delivered on a *`resolution`* frequency. This dataset is created by *`methodology`*. 
+MarketCap by CoinGecko provides marketcap data of the cryptocurrency coins. The dataset covers over 1300 crypto coins that have been listed on CoinGecko website. The data starts in April 2013 and is delivered on a daily frequency.
 
 ### About the Provider
-*`vendorName`* was founded by *`founderName(s)`* in *`foundingYear`*, with the goal of *`missionStatement`*. *`vendorName`* provides access to *`dataTypes`* for *`targetDemographic`*. 
+CoinGecko provides a data platform, digital currency prices, and fundamental analysis of the cryptocurrency market.CoinGecko was founded in 2014 by TM Lee (CEO) and Bobby Ong (COO) with the mission to democratize the access of crypto data and empower users with actionable insights.
 
 ### Getting Started
 Python:
 ```
-# pythonCodeToSubscribeToData
+# Requesting data:
+self.btcusd = self.AddCrypto("BTCUSD", Resolution.Daily).Symbol
```

Comment:
Here we just want to teach how to add data of this dataset, so we don't need to add crypto. This is in line with Louis' comments above.